.gitignore

```diff
@@ -27,7 +27,6 @@ build/Release
 
 # Dependency directory
 node_modules
-typings
```

Comment:
Yep, but sometimes people still downloading typings locally, just wanted to make sure you didnt remove this to commit typings into the repo